src/dyninst/test3_3.C

```diff
@@ -82,7 +82,10 @@ static int readResult(int pid)
 	logerror("ERROR: unable to open output file %s\n", filename);
 	return FAILED;
     }
-    fscanf(fp, "%d\n", &ret);
+    if(fscanf(fp, "%d\n", &ret) == EOF) {
```

Comment:
Do we need the reinterpret_cast here same as what is done in ResumeLog.C?

Reply:
No, `ret` really is an `int`. In ResumeLog.C, it was an enum.